tests/test_polygon.py

```diff
@@ -313,3 +313,22 @@ def test_incircle_from_center(convex_square):
     circle = convex_square.incircle_from_center
     assert np.all(circle.center == convex_square.center)
     assert circle.radius == 0.5
+
+
+@pytest.mark.parametrize("num_sides", range(3, 6))
+def test_perimeter(num_sides):
+    """Test the polygon perimeter calculation."""
+
+    def unit_area_regular_ngon_sidelength(n):
+        r"""Compute the side length of a unit-area regular polygon analytically.
+
+        The area of regular n-gon is given by
+        :math:`\\frac{s^2 n}{4 \\tan\\left(\\frac{180}{n}\\right)}`. This function sets
```

Comment:
raw string, too many slashies ⚔️
```suggestion
        :math:`\frac{s^2 n}{4 \tan\left(\frac{180}{n}\right)}`. This function sets
```